api/test_history_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build medium
```

Comment:
For medium tests, the medium keyword should be specified in the file name, e.g. [labels_medium_test.go](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/labels_medium_test.go)

Reply:
I decided to change it to be a small test. I'm wasn't sure what the differences between test sizes were.